AmericanMedicalAssociation/AMA-style-guide
# Ama 105 primary header organism

https://palantir.atlassian.net/browse/AMA-55

Adds the Primary Header organism with the corresponding atoms and molecules:
Logo atom
Primary Nav link atom
Search Button atom
Primary Nav molecule
Header organism

### Steps to test:

1. Navigate to Atoms > Images > Logo
2. Observe that the AMA logo is present
3. Navigate to Atoms > Links > Link Primary Nav
4. Observe that the link is present with the correct hover state styling.
5. Navigate to Atoms > Buttons > Search Button
6. Observe that the Search Button is present with the correct hover state styling.
7. Navigate to Molecules > Navigation > Primary Navigation
8. Observe that the Primary Navigation links are present with hover states
9. Navigate to Organisms > Global > Header
10. Observe that the complete header is present
